test/KrakenIntegrationTest.cpp

```diff
@@ -100,23 +132,36 @@ TEST_F(KrakenIntegrationTest, VerifyCommunication) {
    // SENDER
    // send all stuff - happy path
    const std::string noError = {"no error"};
-   // send header
+   const size_t kExpectedNumberOfSendActions = 6;
+
+   // signal the async job to exit before the timer if all the
+   // expected data is received
+   expectedToReceive->store(kExpectedNumberOfSendActions);
+   const size_t kraken_header_size = 25;
+
+   // 0. send header
+   LOG(INFO) << "#0 Kraken is sending bytes:  " << header.size() + kraken_header_size;
```

Comment:
I think the logging is overdone here. Is this leftover from debugging why the test wasn't working?